ontobio/io/qc.py

```diff
@@ -352,15 +352,16 @@ def test(self, annotation: association.GoAssociation, config: assocparser.AssocP
 class GoRule16(GoRule):
 
     def __init__(self):
-        super().__init__("GORULE:0000016", "All IC annotations should include a GO ID in the \"With/From\" column", FailMode.SOFT)
+        super().__init__("GORULE:0000016", "All IC annotations should include a GO ID in the \"With/From\" column that is also different from the entry in the \"GO ID\" column", FailMode.SOFT)
 
     def test(self, annotation: association.GoAssociation, config: assocparser.AssocParserConfig, group=None) -> TestResult:
         evidence = str(annotation.evidence.type)
         withfrom = annotation.evidence.with_support_from
+        goId = annotation.object.id
```

Comment:
This is picky but can `goId` be called `go_id` instead to stay consistent with python's underscore [convention](https://peps.python.org/pep-0008/#method-names-and-instance-variables)?

Reply:
Good point.  Will update